demisto_sdk/commands/init/initiator.py

```diff
@@ -118,18 +179,25 @@ def convert_contribution_to_pack(self):
                 with zipped_contrib.open('metadata.json') as metadata_file:
                     click.echo(f'Pulling relevant information from {metadata_file.name}', color=LOG_COLORS.NATIVE)
                     metadata = json.loads(metadata_file.read())
-                    pack_name = metadata.get('name', 'ContributionPack')
+                    # a name passed on the cmd line should take precedence over one pulled
+                    # from contribution metadata
+                    pack_name = self.name or self.convert_to_valid_name(metadata.get('name', 'ContributionPack'))
+                    # a description passed on the cmd line should take precedence over one pulled
+                    # from contribution metadata
+                    pack_description = self.description or metadata.get('description')
```

Comment:
why do we need this in 2 different lines?

Reply:
? I don't understand your comment. Need what in 2 different lines?